integration/samples/scss-paths/sub-module/bar/bar.component.styl

```diff
@@ -0,0 +1,5 @@
+@import "theme"
+@require "common"
```

Comment:
I never used stylus before. Is that common synatx with `@import` and `@require`? Where is the difference between the two?

Reply:
I've not used stylus before either. From what I read they are mostly equivalent for local project imports, so I included examples of both to make sure they both work.

Reply:
FOr documentation on stylus @import and @require: http://stylus-lang.com/docs/import.html